### PR TITLE
Refactoring the code for the local Storage manager

### PR DESCRIPTION
**Pull request by andrayantelo**
Getting rid of the localStorage helper functions in checkit.js. Moving a few to localStorage manager object.

### CODE REVIEW

File: docs/store.js [hunk 1]
```diff
@@ -256,6 +256,23 @@ var LocalCalendarStorage = function(params) {
...
     };
     
+    self.setInStorage = function(key, val) {
+        localstorage.setItem(toKey(key), JSON.stringify(val));
+    };
+    
+    self.getFromStorage = function(key) {
+        var storageItem = localStorage.getItem(toKey(key))
+        if (storageItem !== null) {
+            storageItem = JSON.parse(storagItem);
```
Comment (johntyree) [inline docs/store.js]: Typo

File: docs/store.js [hunk 2]
```diff
@@ -256,6 +256,23 @@ var LocalCalendarStorage = function(params) {
         return key;
     };
     
+    self.setInStorage = function(key, val) {
```
Comment (johntyree) [inline docs/store.js]: These functions shouldn't be on self. No one outside storage should use them

**Review by johntyree — CHANGES_REQUESTED** [review]

File: docs/store.js (hunk 1 above)
Comment (andrayantelo) [reply docs/store.js]: Fixed

File: docs/store.js (hunk 2 above)
Comment (andrayantelo) [reply docs/store.js]: Fixed

**Review by johntyree — APPROVED** [review]

**Pull request merged**